layouts/partials/site-footer.html

```diff
@@ -41,7 +41,7 @@
 
     <img src="/images/gopher-side_color.svg" alt="" class="absolute-l bottom-0 dn db-l h4 right-0 z-999"/>
 
-    <div class="bg-primary-color-dark bottom-0 left-0 right-0 dn-l fixed pb3 ph3 w-100">
+    <div class="bg-primary-color-dark bottom-0 left-0 right-0 dn-l fixed pa3 w-100 flex flex-column" style="max-height: 100%">
```

Comment:
Open to alternatives to the inline `style`, but tachyons doesn't include `max-height` utility classes (i.e. see https://github.com/tachyons-css/tachyons/issues/619).